eng/illink.targets

```diff
@@ -1,5 +1,7 @@
 <Project>
 
+  <Import Project="$(ToolsILLinkDir)$(NetCoreAppToolCurrent)/build/Microsoft.NET.ILLink.Tasks.props" />
```

Comment:
It's important to only import files that exist at msbuild evaluation time. Where does that file come from?

Reply:
Can you import it from the checked-in place instead? https://github.com/dotnet/runtime/blob/main/src/tools/illink/src/ILLink.Tasks/build/Microsoft.NET.ILLink.Tasks.props. Btw it's a weird for a targets file to import a props file. We might want to clean that up eventually when consolidating the stuff in illink.targets with Microsoft.NET.ILLink.Tasks.targets.